src/Paket.PowerShell/System.Management.Automation.ps1

```diff
@@ -0,0 +1,2 @@
+# copies System.Management.Automation.dll from the GAC to this folder
+cp ([PSObject].Assembly.Location) .
```

Comment:
![image](https://cloud.githubusercontent.com/assets/57396/8143528/a9cd49a4-11b1-11e5-8f36-c522961fe11a.png)

@latkin doesn't seem to work?!